geode-core/src/distributedTest/java/org/apache/geode/internal/cache/eviction/EvictionDUnitTest.java

```diff
@@ -256,8 +241,13 @@ private static int getExpectedEviction() {
       hmm.updateStateAndSendEvent(90);
     }
 
-    return (int) Math
-        .ceil((evictor.getTotalBytesToEvict() / 2) / (double) ((1024 * 1024) + 100)) * 2;
+    int entrySize = ENTRY_SIZE + 100;
+    long totalBytesToEvict = evictor.getTotalBytesToEvict();
+    int expectedEviction = (int) Math.ceil((double) totalBytesToEvict / (double) entrySize);
+
+    GeodeAwaitility.await()
+        .until(() -> (abs(region.getTotalEvictions() - expectedEviction) <= 1));
```

Comment:
Won't this always be true?  If expectedEviction is 1 and totalEvictions is 0 then subtracting 0-1 and taking the abs will always be <= 1?

If expected is 10 and total is 0, then again it's bigger than 1?
My math brain isn't quite on this morning so I might be totally off...

Reply:
yeah, I think since the eviction happens on another thread, this awaitility is there only to wait till we are close to examine the actual evictions that has happened.